### PR TITLE
chore: Allow WPT runner only to update WPT expectations

### PR DESCRIPTION
**Pull request by sadym-chromium**
Required for sharding WPT tests.
* Add `RUN_TESTS` env with default `true`;
* Extend help message.

### CODE REVIEW

File: tools/run-wpt.mjs [hunk 1]
```diff
@@ -90,87 +103,98 @@ if (HEADLESS === 'true') {
   log('Running WPT in headful mode...');
 }
 
-const wptRunArgs = [
```
Comment (sadym-chromium) [inline tools/run-wpt.mjs:93]: This change block looks scary, but in fact I only moved running logic to `if (RUN_TESTS === 'true')`.

File: tools/run-wpt.mjs [hunk 2]
```diff
@@ -32,7 +32,16 @@ function log(message) {
...
+    `Usage:
+      [CHROMEDRIVER=<true | default: false>]
+      [HEADLESS=<true | default: false>]
+      [MANIFEST=<default: 'MANIFEST.json'>]
+      [RUN_TESTS=<default: true | false>]
+      [UPDATE_EXPECTATIONS=<true | default: false>]
+      [VERBOSE==<true | default: false>]
+      [WPT_REPORT=<default: 'wptreport.json'>]
+      [WPT_METADATA=<default: 'wpt-metadata/$\{ 'chromedriver' | 'mapper' }/$\{ 'headless' | 'headful' }>]
+      ${process.argv[1]} [webdriver/tests/bidi/[...]]`
```
Comment (sadym-chromium) [inline tools/run-wpt.mjs:44]: @jrandolf I'm not sure about the help format, but it's not critical. If you think it should be written in another way, feel free to add edit suggestion.

Comment (sadym-chromium) [reply tools/run-wpt.mjs:44]: This version looks like:
```
      [CHROMEDRIVER=<true | default: false>]
      [HEADLESS=<true | default: false>]
      [MANIFEST=<default: 'MANIFEST.json'>]
      [RUN_TESTS=<default: true | false>]
      [UPDATE_EXPECTATIONS=<true | default: false>]
      [VERBOSE==<true | default: false>]
      [WPT_REPORT=<default: 'wptreport.json'>]
      [WPT_METADATA=<default: 'wpt-metadata/${ 'chromedriver' | 'mapper' }/${ 'headless' | 'headful' }>]
      chromium-bidi/tools/run-wpt.mjs [webdriver/tests/bidi/[...]]
```

File: tools/run-wpt.mjs [hunk 3]
```diff
@@ -90,87 +103,98 @@ if (HEADLESS === 'true') {
...
-    '--install-webdriver',
-    `--webdriver-arg=--bidi-mapper-path=${join('lib', 'iife', 'mapperTab.js')}`,
-    `--webdriver-arg=--log-path=${join('out', 'chromedriver.log')}`,
-    `--webdriver-arg=--log-level=${VERBOSE === 'true' ? 'ALL' : 'INFO'}`,
-    '--yes'
-  );
-} else {
-  wptRunArgs.push(
+let run_status = undefined,
+  update_status = undefined;
```
Comment (Lightning00Blade) [inline tools/run-wpt.mjs:109]:
```suggestion
let run_status = undefined;
let update_status = undefined;
```

File: tools/run-wpt.mjs [hunk 4]
```diff
@@ -185,7 +209,15 @@ if (UPDATE_EXPECTATIONS === 'true') {
...
     ],
     {stdio: 'inherit'}
-  );
+  ).status;
+}
+
+// If WPT tests themselves or the expectations update failed, return failure.
+let result_status = 0;
+if ((run_status ?? 0) !== 0) {
+  result_status = run_status;
```
Comment (Lightning00Blade) [inline tools/run-wpt.mjs:218]: Let's add log in each if to see easily see which failed.

**Review by Lightning00Blade — APPROVED** [review]
LGTM + some minor comments

**Pull request merged**